<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabGraph">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>292</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelIP">
           <property name="text">
            <string>IP:Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditIP">
           <property name="text">
            <string>127.0.0.1:7777</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Listen">
           <property name="text">
            <string>Listen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QCheckBox" name="checkBox_RawData">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Raw data</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_Clear">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Clear Graph, log when start listening</string>
        </property>
        <property name="text">
         <string>Clear graph, log</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
